config/spring19_seminar/cola_gpt.conf

```diff
@@ -8,6 +8,7 @@ include "cola_elmo.conf"
 do_target_task_training = 0  // only do main training phase, since our pretrain and target task are the same
 allow_reuse_of_pretraining_parameters = 1  // set this to 1 because we pretrain and finetune model on the same task
 batch_size = 2  // keep it small to avoid out-of-memory
+lr = 1e-5  // we need to keep lr small for fine-tuning gpt, especially so because our BS is smaller than that in gpt paper
```

Comment:
Fine to check this in, but we probably won't be publishing anything with GPT, right?

Reply:
No, we won't